openprescribing/measures/definitions/environmental_inhalers.json

```diff
@@ -1,17 +1,25 @@
 {
-  "name": "Environmental impact of inhalers - Prescribing of the less environmentally friendly Metered Dose Inhaler (MDI) (excluding salbutamol)",
+  "name": "Environmental impact of inhalers - Prescribing of Metered Dose Inhaler (MDI) (excluding salbutamol)",
```

Comment:
The "the less environmentally friendly" part has been removed from the title - this was added earlier this year to help clarify the measure a bit better. Is there a reason to remove it?